en/config_fw/advanced_tuning_guide_fixedwing.md

```diff
@@ -79,9 +79,9 @@ Furthermore, these two values define the height rate limits commanded by the use
 
 ### L1 Controller Tuning (Position)
```

Comment:
@sfuhrer Can you propose the docs you'd like to see here? 

Based on your other fix it is the stuff here http://docs.px4.io/main/en/advanced_config/parameter_reference.html#fw-npfg-control

What is NPFG?

Reply:
PS Note, I have tagged bits that need your attention. Ignore anything not tagged to you.

Reply:
I'd like to leave that to @tstastny . NPFG is the new FW guidance that replaced the L1 controller. How to tune it stays largely the same though I would say.

Reply:
@tstastny  I'm merging this PR because it fixes lots of problems. Can you create a separate PR to address this specific section?